IOPool/Common/test/BuildFile.xml

```diff
@@ -1,14 +1,3 @@
-<environment>
-  <bin file="TestIOPoolCommon.cpp" name="TestEdmFastMerge">
```

Comment:
Could the `TestIOPoolCommon.cpp` be removed as well?

Reply:
yes